### DISCUSSION TITLE
Region election

### DISCUSSION THREAD

**Discussion by a1120960** [hadynz/obsidian-kindle-plugin#249] [General]
Hi, is there any way to add a new region? I'm in Australia and its not available from the drop sown list in the settings

Thanks
B

**Comment by hadynz** [comment]
Australia _should_ just work using the global (`.com`) setting.

Have you tried pulling your highlights through the plugin yet?

**Reply by a1120960** [reply]
ah yes, thank you. On first attempt with the global region it just timed out, after restarting it works perfectly. Thanks!
